src/databricks/labs/ucx/cli.py

```diff
@@ -138,6 +139,37 @@ def repair_run(w: WorkspaceClient, step):
     installer.repair_run(step)
 
 
+@ucx.command
+def validate_groups_membership(w: WorkspaceClient):
```

Comment:
What if I wan't to scope it for all workspaces in my account ?

Reply:
It has to be an additional command